app/views/users/_name.html.erb

```diff
@@ -1,2 +1,4 @@
-<tr><td><%= label_tag('user[fullname]', 'Full name (last, first[ middle]): ') %></td><td>
-  <%= text_field 'user', 'fullname', class: "form-control width-250"%></td></tr>
+<tr>
+  <td><label class="label_account_request">Full name (last, first [middle]): </label></td>
+  <td><%= text_field 'user', 'fullname', class: "form-control width-250"%></td>
+</tr>
```

Comment:
Please add a new line at the end of this file for consistent formatting (please also do this at the end of any other file that's missing this)

Reply:
I will fix this in the next update